demo/integration.js

```diff
@@ -139,17 +140,17 @@ export function createAndInit() {
 
 function getCollections() {
     return [
+        {
```

Comment:
>Default dictionary should be **only** PRECISE

Given the presence of only in the phrase above I wonder if the customer is not asking to remove the default search in data dictionary fields instead of just changing the default collection from Interlex to Precise.

![image](https://github.com/MetaCell/cde-mapper/assets/19196034/cd00bb7f-4759-4562-9661-f99334de0a31)
![image](https://github.com/MetaCell/cde-mapper/assets/19196034/cd00bb7f-4759-4562-9661-f99334de0a31)

Reply:
that's customisable in the integration tbh, I am up to swap the order and change the default but I want to leave to them as many references as possible in the integration to make their life easier, so I am not removing it now. Thx